src/main/java/ui/UI.java

```diff
@@ -268,7 +268,10 @@ private boolean isCloseOnQuit() {
         return commandLineArgs.getOrDefault("closeonquit", "false").equalsIgnoreCase("true");
     }
 
-    public void quit() {
+    public void quit(boolean isLogout) {
```

Comment:
What I was thinking was

```java
class UI {
    void logout() {
        browserComponent.cleanChromeCustomProfile();
        quit();
    }
    void quit() {
        browserComponent.onAppQuit();
        // whatever else is here
    }
}
```
It's a minor thing though, just cosmetic. The UI mostly just interfaces directly with the browser component so there's not much else involved.